recipes/glib/all/test_package/conanfile.py

```diff
@@ -8,7 +8,9 @@ class TestPackageConan(ConanFile):
 
     def build(self):
         if self.settings.os != "Windows":
-            self.run("gdbus-codegen -h", run_environment=True)
+            with tools.environment_append({'PKG_CONFIG_PATH': "."}):
```

Comment:
This test should move to `test` because it does not build anything.
Also, it should be conditionally run when not cross building.

Reply:
This is the same as the other PR. In a client recipe, this tool is invoked during the build

Reply:
When we are cross building glib, won't executing `gdbug-codegen` fail?

Reply:
Maybe it fails, but that would be a bug

Reply:
It will work if it's a shell/python script. Not if it is a binary.